src/main/kotlin/nexters/admin/domain/user/administrator/Administrator.kt

```diff
@@ -14,9 +14,17 @@ class Administrator(
         var password: Password,
 
         @Column(name = "last_access_time")
-        var lastAccessTime: LocalDateTime? = null
+        var lastAccessTime: LocalDateTime? = null,
 ) {
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     var id: Long = 0L
+
+    fun isSamePassword(password: Password): Boolean {
+        return this.password.value == password.value
```

Comment:
테스트 코드의 중복(관리자, 일반회원에서의 검증 테스트)이 발생할 것 같은데`Password` VO한테 메시지를 던지도록 변경해보는 것도 괜찮겠네요. 패스워드가 일치하는지 검증의 책임은 패스워드한테 맡겨도 괜찮을 듯해요

```suggestion
        return this.password.isSamePassword(password)
```

Reply:
아 네 좋네요.